cards/faq-accordion/component.js

```diff
@@ -76,18 +76,24 @@ class faq_accordionCardComponent extends BaseCard['faq-accordion'] {
 
     if (stayExpanded && this.getState('feedbackSubmitted')) {
       isExpanded = true;
-      cardEl.classList.toggle(accordionExpandedClass, isExpanded);
+      if (!cardEl.classList.contains(accordionExpandedClass)) {
+        cardEl.classList.toggle(accordionExpandedClass, isExpanded);
+      }
       accordionToggleEl.setAttribute('aria-expanded', isExpanded ? 'true' : 'false');
     }
-    cardEl.classList.toggle(stayExpandedClass, false);
+    if (cardEl.classList.contains(stayExpandedClass)) {
+      cardEl.classList.toggle(stayExpandedClass, false);
+    }
     contentEl.style.height = `${isExpanded ? contentEl.scrollHeight : 0}px`;
     this._setLinksInteractivity(linkEls, isExpanded);
 
     const thumbSelectorEls = this._container.querySelectorAll('.js-HitchhikerCard-thumbInput');
     if (thumbSelectorEls) {
       thumbSelectorEls.forEach(el => {
         el.addEventListener('click', (e) => {
-          cardEl.classList.toggle(stayExpandedClass, true);
+          if (!cardEl.classList.contains(stayExpandedClass)) {
```

Comment:
similarly, this always get added when clicking the feedback button, so I think just having `cardEl.classList.add(stayExpandedClass);` without if statement should be good!